ui/src/main/webapp/src/app/executions/execution-detail.component.html

```diff
@@ -47,6 +47,10 @@ <h3 i18n="Analysis status|Section">Status</h3>
                     <dt i18n="Analysis last modification">Last modification</dt>
                     <dd>{{execution.lastModified ? (execution.lastModified | date: 'short') : 'n/a'}}</dd>
                 </ng-container>
+                <ng-container *ngIf="execution.state == 'COMPLETED' && execution?.analysisContext?.generateStaticReports">
+                    <dt i18n="Analysis static report">Report</dt>
+                    <dd><a class="link" target="_blank" href="{{formatStaticReportUrl(execution)}}">Static Report <i class="fa fa-external-link"></i></a></dd>
```

Comment:
Ok, i tried them all also with a third solution of having just the row header "Report" with the link.
As is customary, i upload a screenshot with the 3 solutions:
![screenshot from 2017-06-27 10-32-14](https://user-images.githubusercontent.com/7288588/27578419-4efe3bd2-5b24-11e7-83a5-19c2752e27a0.png)

+1 for the third and last solution